CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to the "vscode-gradle" extension will be documented in this
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 3.8.3
```

Comment:
@CsCherrYY The release date should be added after the release version number, which is easy to make people have a sense of time.

Reply:
PR created:https://github.com/microsoft/vscode-gradle/pull/1102